plugins/com.google.cloud.tools.eclipse.appengine.validation/xslt/servlet.xsl

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+	This stylesheet changes the servlet in web.xml to servlet 2.5.
+-->
+<xsl:stylesheet version="1.0"
+	xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+  <xsl:template match="node()|@*">
+    <xsl:copy>
+      <xsl:apply-templates select="node()|@*"/>
+    </xsl:copy>
+  </xsl:template>
+   
+  <xsl:template match="/*">
```

Comment:
I think you can match a specific element here. I.e.

web-app  in the namespace http://xmlns.jcp.org/xml/ns/javaee